plugins/inputs/dns_query/README.md

```diff
@@ -33,8 +33,14 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## Dns server port.
   # port = 53
 
-  ## Query timeout in seconds.
-  # timeout = 2
+  ## Query timeout
+  # timeout = "2s"
+
+  ## Include the specified additional properties in the resulting metric.
+  ## The following values are supported:
+  ##    IP        -- return IP of the first A and AAAA answer
+  ##    "all IPs" -- return IP of the all A and AAAA answers with index
```

Comment:
It feels weird to see an option with mixed case + space and both should really be quoted, what do you think about:

```suggestion
  ##    "first_ip"    -- return IP of the first A and AAAA answer
  ##    "all_ips"     -- return IPs of the all A and AAAA answers with index
```